api/user/followers/followers.py

```diff
@@ -101,9 +98,9 @@ def followers_count(author_id: str):
 @basic_auth.required
 def following_count(author_id: str):
     """Get the count for the number of people author is following"""
-    following = Author.query.filter_by(id=author_id).join(follows_table, follows_table.c.follower_id == Author.id).all()
-    non_local_following = NonLocalFollower.query.filter_by(follower_id=author_id).all()
-    return {"count": len(following) + len(non_local_following)}, 200
+    following = LocalFollower.query.filter_by(follower_id=author_id).count()
+    non_local_following = NonLocalFollower.query.filter_by(follower_id=author_id).count()
```

Comment:
NonLocalFollower table has followed_id column as foreignkey from author.id and Author table doesnt have foreign authors in it. I think we need another table for NonLocalFollowing

Reply:
Check model.py for followers